pages/common/multitail.md

```diff
@@ -1,7 +1,7 @@
 # multitail
 
 > Extension of tail.
-> More information: <https://www.vanheusden.com/multitail/examples.php>.
+> More information: <https://manned.org/multitail>.
```

Comment:
Strange.  The man page has this very URL, but, besides being very insecure, is a 404.

Reply:
Seems like <vanheusden.com> was repurposed. It seems to be something historical now.